samples/scan_bm/src/main.c

```diff
@@ -101,7 +101,7 @@ int main(void)
 
 	printf("WiFi scan sample application using nRF70 Bare Metal library\n");
 
-	memcpy(reg_info.country_code, "IN", 2);
```

Comment:
@krish2718 IN stands for International, or?

Reply:
`INDIA` :)